openpype/hosts/houdini/api/lib.py

```diff
@@ -747,3 +752,32 @@ def get_camera_from_container(container):
 
     assert len(cameras) == 1, "Camera instance must have only one camera"
     return cameras[0]
+
+
+def validate_job_path():
+    """Validate job path to ensure it matches the settings."""
+
+    project_settings = get_current_project_settings()
+
+    if project_settings["houdini"]["general"]["job_path"]["enabled"]:
+
+        # get and resolve job path template
+        job_path_template = \
+            project_settings["houdini"]["general"]["job_path"]["path"]
+        job_path = StringTemplate.format_template(
+            job_path_template, get_current_context_template_data()
+        )
+        job_path = job_path.replace("\\", "/")
+
+        if job_path == "":
+            # Set JOB path to HIP path if JOB path is enabled
+            # and has empty value.
+            job_path = os.environ["HIP"]
+
+        current_job = hou.hscript("echo -n `$JOB`")[0]
+        if current_job != job_path:
+            hou.hscript("set JOB=" + job_path)
+            os.environ["JOB"] = job_path
+            print("  - set $JOB to " + job_path)
+    else:
+        print("  - JOB Path is disabled, Skipping Check...")
```

Comment:
Shouldn't this be `logging` calls instead?

Reply:
yes, but for some reason logging doesn't work, so I chose print. 
same problem found here 
https://github.com/ynput/OpenPype/blob/1bd07bd15b1bf238e186b23b7d112e9fe4637737/openpype/hosts/houdini/api/lib.py#L505

Reply:
it is supposed that you can simply type `log.debug("Setting scene FPS to {}".format(int(fps)))` to do the logging calls as the logging system is implemented in https://github.com/ynput/OpenPype/blob/fdea715fe0d7aafd7f3000b8aca7780d432aeacb/openpype/hosts/houdini/api/lib.py#L26

Reply:
let's just use `print` and we can change it when log works